eng/scripts/determine-js-release-tag.ps1

```diff
@@ -0,0 +1,18 @@
+# Read the package.json file
+$packageJsonPath = "$PSScriptRoot/../../package.json"
+$packageJson = Get-Content $packageJsonPath -Raw | ConvertFrom-Json
+
+# Function to check if a version is non-GA
```

Comment:
Is it worth using our SemVer.ps1 here? Do we have non-standard prerelease tags?

Reply:
I'll take a look, thanks for the suggestion 👍

Reply:
Yes, I'll parse the version from package.json, then set `beta` if isPrerelease` is true`.